{{cookiecutter.repo_slug}}/.github/labels.yml

```diff
@@ -13,10 +13,10 @@
   description: "Enhancement of the code, not introducing new features."
 - name: "refactor"
   color: 1d76db
-  description: "Improvement of existing code, not introducing new features."
+  description: "Updating the code with simpler, easier to understand or more efficient syntax or methods, but not introducing new features."
```

Comment:
I actually don't know what the difference between "Enhancement of the code" (previous item) and "Improvement of the code" is supposed to be. Many projects seem to use the "enhancement" label as a stand-in for "new feature." If you're going to suggest that it just be about the code (so that you can have a more explicit new-feature label, which I agree with) then these two need to be distinguished somehow, or one of them removed. So my suggested change gives some scope to the "refactor" label based on what refactoring generally includes, but it doesn't address "enhancement" which to me remains vague.